packages/react-core-notifications/src/InAppMessaging/hooks/useMessage/useMessage.ts

```diff
@@ -40,26 +36,17 @@ export default function useMessage<PlatformStyleProps>({
   const { content, layout } = message;
 
   const onActionCallback = () => {
-    InAppMessaging.notifyMessageInteraction(
-      message,
-      InAppMessageInteractionEvent.MESSAGE_ACTION_TAKEN
-    );
+    notifyMessageInteraction({ message, type: 'messageActionTaken' });
     clearMessage();
   };
 
   const onClose = () => {
-    InAppMessaging.notifyMessageInteraction(
-      message,
-      InAppMessageInteractionEvent.MESSAGE_DISMISSED
-    );
+    notifyMessageInteraction({ message, type: 'messageDismissed' });
     clearMessage();
   };
 
   const onDisplay = () => {
-    InAppMessaging.notifyMessageInteraction(
-      message,
-      InAppMessageInteractionEvent.MESSAGE_DISPLAYED
-    );
+    notifyMessageInteraction({ message, type: 'messageDisplayed' });
```

Comment:
Formatting looks off here, might want to check if prettier is configured to run on save in your editor